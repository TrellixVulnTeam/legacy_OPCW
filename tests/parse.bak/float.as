# Various floating-point numbers.
43.43626;
3246.3427237;
345.4327237;
3242e329;
23E2;
42.234E32;
324.3e32;
