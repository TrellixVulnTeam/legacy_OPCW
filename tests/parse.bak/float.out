ModuleDecl <?> _
  Nodes <?> 
    FloatExpr <?> 43.43626
    FloatExpr <?> 3246.3427237
    FloatExpr <?> 345.4327237
    FloatExpr <?> 3242e329
    FloatExpr <?> 23E2
    FloatExpr <?> 42.234E32
    FloatExpr <?> 324.3e32
